rflx/generator/session.py

```diff
@@ -324,7 +324,7 @@ def _create_function_parameters(
                     if a.type_ == rty.OPAQUE
                     else ID(self._prefix * a.type_identifier * "Structure")
                     if isinstance(a.type_, rty.Message)
-                    else ID(self._prefix * a.type_identifier),
+                    else ID(a.type_identifier),
```

Comment:
The removed prefix will lead to an error when the function parameter or return type has the same name as the package (e.g., renaming `My_Int` to `Empty_Ident` in the added test case).